network-store-server/src/main/resources/db/changelog/changesets/twt_regulationPoint_20241126T110000Z.sql

```diff
@@ -0,0 +1,66 @@
+from twowindingstransformer;
+
+-- three windings transformer
+-- side 1
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_ONE'), 'PHASE_TAP_CHANGER', phasetapchangerregulationmode1, id, 'ONE', phasetapchangerterminalrefconnectableid1, phasetapchangerterminalrefside1
+from threewindingstransformer;
+
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_ONE'), 'RATIO_TAP_CHANGER', ratiotapchangerregulationmode1, id, 'THREE', ratiotapchangerterminalrefconnectableid1, ratiotapchangerterminalrefside1
+from threewindingstransformer;
+-- side 2
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_TWO'), 'PHASE_TAP_CHANGER', phasetapchangerregulationmode2, id, 'THREE', phasetapchangerterminalrefconnectableid2, phasetapchangerterminalrefside2
+from threewindingstransformer;
+
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_TWO'), 'RATIO_TAP_CHANGER', ratiotapchangerregulationmode2, id, 'TWO', ratiotapchangerterminalrefconnectableid2, ratiotapchangerterminalrefside2
+from threewindingstransformer;
+-- side 3
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_THREE'), 'PHASE_TAP_CHANGER', phasetapchangerregulationmode3, id, 'THREE', phasetapchangerterminalrefconnectableid3, phasetapchangerterminalrefside3
+from threewindingstransformer;
+
+insert into regulatingpoint (networkuuid, variantnum, regulatingequipmentid, regulatingequipmenttype, regulationmode, localterminalconnectableid, localterminalside, regulatingterminalconnectableid, regulatingterminalside)
+select networkuuid, variantnum, concat(id, '_THREE'), 'RATIO_TAP_CHANGER', ratiotapchangerregulationmode3, id, 'THREE', ratiotapchangerterminalrefconnectableid3, ratiotapchangerterminalrefside3
+from threewindingstransformer;
+UPDATE regulatingpoint r1
+SET regulatedequipmenttype = (select CASE
```

Comment:
Why do we need to go through all these cases again?

Reply:
to get the type of the selected equipment 
to optimize request